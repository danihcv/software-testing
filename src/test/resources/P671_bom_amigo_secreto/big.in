25
5
738
111
213
987
69
73
741
123
456
789
325
542
999
666
1000
264
7
544
459
969
679
258
333
99