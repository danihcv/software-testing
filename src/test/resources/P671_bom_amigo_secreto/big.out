12
452856785
324852642
808306982
625460659
551978127
759229837
667127729
820427921
92890357
675633517
2320477
817904102
511389714
351702175
878320716
92932378
360
82958214
70321864
934308881
922985407
734681872
804763448
495476666
1
